### ISSUE TITLE
SQL share

### ISSUE THREAD

**Issue by lucasvperini** [oMarkez/FSH_GlobalBans#2]
Hey!

Very good script.
Would you mind on sharing database? 
I want to host my own global bans, if possible.

Thanks!

**Comment by oMarkez** [comment]
No, I'm not sharing the database. Althought you can create your own, for your own global bans, just remember to create a user with select-only perms for that database.

If you have any proof of a modder with a steam/gta id, then you can send it to me and I'll globally ban the person from all the vRP servers using my resource. Thanks :)

**Comment by lucasvperini** [comment]
Not the database, just the SQL schema, my misspell.

(Sorry for the bad english, i'm Brazilian).

**Comment by oMarkez** [comment]
Ah yes of course, here:

SET` SQL_MODE = "NO_AUTO_VALUE_ON_ZERO";
SET time_zone = "+00:00";

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!40101 SET NAMES utf8mb4 */;


CREATE TABLE `bans` (
  `banid` int(11) NOT NULL,
  `steamid` varchar(50) NOT NULL,
  `gtalicense` varchar(50) NOT NULL,
  `bangrund` mediumtext NOT NULL,
  `timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP
) ENGINE=InnoDB DEFAULT CHARSET=latin1;


ALTER TABLE `bans`
  ADD PRIMARY KEY (`banid`);


ALTER TABLE `bans`
  MODIFY `banid` int(11) NOT NULL AUTO_INCREMENT;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;

**Comment by lucasvperini** [comment]
Thank you!
